src/main/scala/strawman/collection/StringOps.scala

```diff
@@ -10,11 +10,14 @@ import scala.reflect.ClassTag
 
 class StringOps(val s: String)
   extends AnyVal
-     with SeqOps[Char, Seq, String]
-     with Buildable[Char, String]
-     with ArrayLike[Char] {
+    with IterableOnce[Char]
```

Comment:
To implement `IterableOnce` we only need to implement the `iterator()` method, which is already present in `StringOps`. Said otherwise, extending `IterableOnce` is “free”.